src/dash/Dashboard.jsx

```diff
@@ -2,20 +2,20 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import HomeMenuBar from '../MenuBar.jsx';
 import DefaultDash from './DefaultDash.jsx';
+import DefaultProfile from './Profile.jsx';
 import { Provider } from 'react-redux';
 import store from '../redux/store.js';
 import AboutUs from '../home/AboutUs.jsx';
 import {Router, Route, browserHistory} from 'react-router';
 
-
-
 const DashComp = function() {
   return (
     <div>
       <HomeMenuBar homeordash='dash'/>
       <Router history={browserHistory}>
           <Route path="/dash" component={DefaultDash}/>
```

Comment:
I believe this will render the DefaultDash component every time because the `/dash` route will also match with `/dash/about` and `/dash/profile`. Is this intended?